src/entities/liquidity-mining-campaign.ts

```diff
@@ -22,13 +22,13 @@ const MINIMUM_STAKED_AMOUNT_NATIVE_CURRENCY: { [chainId in ChainId]: CurrencyAmo
     utils.parseUnits('1000', Token.getNative(ChainId.XDAI).decimals).toString(),
     ChainId.XDAI
   ),
-  [ChainId.SOKOL]: CurrencyAmount.nativeCurrency(
-    utils.parseUnits('1000', Token.getNative(ChainId.SOKOL).decimals).toString(),
-    ChainId.SOKOL
+  [ChainId.ARBITRUM_ONE]: CurrencyAmount.nativeCurrency(
+    utils.parseUnits('0.1', Token.getNative(ChainId.ARBITRUM_ONE).decimals).toString(),
```

Comment:
What determines the value passed in here?

Reply:
It's pretty arbitrary tbh. We can increase it if we think the floor is too low atm.